etc/openstack_deploy/group_vars/all/osa.yml

```diff
@@ -64,6 +64,21 @@ glance_glance_api_conf_overrides:
   DEFAULT:
     image_cache_dir: None
 
+# Octavia tuning
+# activate octavia - Queens only
+octavia_v2: True
+octavia_v1: False
+# Keep this while there is no Barbican or Vault
+octavia_tls_listener_enabled: False
+
+# Octavia HA settings
+octavia_loadbalancer_topology: ACTIVE_STANDBY
+octavia_spare_amphora_pool_size: 0
+octavia_enable_anti_affinity: {{ lookup('env', 'DEPLOY_AIO') != "yes" }}
```

Comment:
change "yes" to 'yes' and wrap {{ lookup('env', 'DEPLOY_AIO') != "yes" }} in double quotes.